### DISCUSSION TITLE
First beta release of v1.0.0

### DISCUSSION THREAD

**Discussion by HJfod** [geode-sdk/sdk#113] [Announcements]
Introducing a major redesign of many parts of the framework, including [a new hooking framework](https://github.com/geode-sdk/TulipHook), [a string ID & layout system for nodes](https://docs.geode-sdk.org/#/tutorials/nodetree?id=string-ids), improvement of many existing features, and much more.

The purpose of this release is to make sure the framework is production-ready, and to catch bugs / missing features before the full release of v1.0.0 while we are working on documentation ([we've got a new site!](https://geode-sdk.org)). Once the full release of v1.0.0 hits, the API will be stabilized - from there on out, we will be following semver, and you can be quite confident that the mods you write now will work in the future, without needing to update them every few weeks.

**If you encounter any issues using Geode** (installation, documentation, usage, etc.), **please do let us know** through [Github issues](https://github.com/geode-sdk/geode/issues) or through [our Discord server](https://discord.gg/9e43WMKzhp).

<hr /><em>This discussion was created from the release <a href='https://github.com/geode-sdk/geode/releases/tag/v1.0.0-beta0'>First beta release of v1.0.0</a>.</em>

**Comment by TechStudent10** [comment]
Lets go! Can't wait to get started!
